samples/subsys/event_manager/prj.conf

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Enabling assert
+CONFIG_ASSERT=y
+
+# Logger configuration
+CONFIG_LOG=y
+CONFIG_LOG_DEFAULT_LEVEL=2
+CONFIG_LOG_BACKEND_UART=y
```

Comment:
can be removed, on most platforms enabling logging comes with default backend and for some it's not uart.